modules/covid_vaccine/app/models/covid_vaccine/v0/expanded_registration_submission.rb

```diff
@@ -19,6 +50,54 @@ class ExpandedRegistrationSubmission < ApplicationRecord
       attr_encrypted :raw_form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
       attr_encrypted :eligibility_info, key: Settings.db_encryption_key, marshal: true,
                                         marshaler: JsonMarshal::Marshaller
+
+      # We want records that have acceptable discharge status, country, state, zip and facility
+      def self.to_csv(separater = '^')
+        CSV.generate(col_sep: separater) do |csv|
+          eligible_us.order('created_at DESC').each do |es|
+            csv << es.send(:csv_row)
+          end
+        end
+      end
+
+      private
+
+      def country_us?
+        raw_form_data[:country_name] == 'USA'
+      end
+
+      def csv_row
```

Comment:
I'm wondering if we want to run all fields through a sanitizer function that strips `^` and also `"` (which Mehedi mentioned was disallowed).

Reply:
I thought about that too -- could have a sanitize function on csv_row or at the very end when the entire string is created and just prior to sending.

Also not entirely married to this code living here in this class, but it works for now.